meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/start-resin-supervisor

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+source /usr/sbin/resin-vars
+
+SUPERVISOR_IMAGE_ID=$(docker inspect $SUPERVISOR_IMAGE | jq '.[0].Id')
```

Comment:
Good idea, done.
(It is indeed [supported in 1.10.3](https://github.com/docker/docker/blob/v1.10.3/docs/reference/commandline/inspect.md))